packages/flutter_tools/test/general.shard/web/devfs_web_ddc_modules_test.dart

```diff
@@ -117,1385 +62,1214 @@ void main() {
     httpServer = FakeHttpServer();
     linux = FakePlatform(environment: <String, String>{});
     windows = FakePlatform(
-        operatingSystem: 'windows', environment: <String, String>{});
+      operatingSystem: 'windows',
+      environment: <String, String>{},
+    );
     logger = BufferLogger.test();
     testbed = Testbed(
-        setup: () {
-          webAssetServer = WebAssetServer(
-            httpServer,
-            packages,
-            InternetAddress.loopbackIPv4,
-            <String, String>{},
-            <String, String>{},
-            NullSafetyMode.unsound,
-            usesDdcModuleSystem,
-            webRenderer: WebRendererMode.canvaskit,
-            useLocalCanvasKit: false,
-          );
-          releaseAssetServer = ReleaseAssetServer(
-            globals.fs.file('main.dart').uri,
-            fileSystem: globals.fs,
-            flutterRoot: null, // ignore: avoid_redundant_argument_values
-            platform: FakePlatform(),
-            webBuildDirectory: null, // ignore: avoid_redundant_argument_values
-            needsCoopCoep: false,
-          );
+      setup: () {
+        webAssetServer = WebAssetServer(
+          httpServer,
+          packages,
+          InternetAddress.loopbackIPv4,
+          <String, String>{},
+          <String, String>{},
+          NullSafetyMode.unsound,
+          usesDdcModuleSystem,
+          webRenderer: WebRendererMode.canvaskit,
+          useLocalCanvasKit: false,
+        );
+        releaseAssetServer = ReleaseAssetServer(
+          globals.fs.file('main.dart').uri,
+          fileSystem: globals.fs,
+          flutterRoot: null,
+          platform: FakePlatform(),
+          webBuildDirectory: null,
+          needsCoopCoep: false,
+        );
+      },
+      overrides: <Type, Generator>{Logger: () => logger},
+    );
+  });
+
+  @isTest
+  void runInTestbed(
```

Comment:
nit: Should this be private? Alternatively, it could be moved to wherever the `Testbed` class is defined if we want this to be reusable outside of the class (I'm not terribly concerned about this, because—if I recall correctly—we'd like to remove the `Testbed` construct entirely).

```suggestion
  void _runInTestbed(
```

Reply:
I think my preference would be to keep this public but move it to the testbed.dart file.

Reply:
Fortunately, it's already private in a sense, given that it's a local function defined inside `main() {}`.

I don't think it's able to be moved though, since it references the `late Testbed testbed` local variable.

Reply:
> Fortunately, it's already private in a sense, given that it's a local function defined inside `main() {}`.

Oh, I failed to notice this was scoped to `main`. No worries here then.

> I don't think it's able to be moved though, since it references the `late Testbed testbed` local variable.

Gotcha.